tasks/seal/task_submit_commit.go

```diff
@@ -248,7 +251,12 @@ func (s *SubmitCommitTask) Do(taskID harmonytask.TaskID, stillOwned func() bool)
 		return false, xerrors.Errorf("could not serialize commit params: %w", err)
 	}
 
-	collateral, err := s.api.StateMinerInitialPledgeCollateral(ctx, maddr, pci.Info, ts.Key())
```

Comment:
Waait how did that work with filplus deals? It couldn't be correct, right?

Reply:
I don't think it did. We were sending incorrect collateral AFAIK.